randovania/games/dread/json_data/Dairon.txt

```diff
@@ -1603,78 +1411,34 @@ Extra - asset_id: collision_camera_018
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
   > Dock to Big Hub
-      Morph Ball and After Dairon - Central Unit
+      Any of the following:
+          Flash Shift or Flash Shift Skip (Beginner) or Lay Cross Bomb or Use Spin Boost
+          Morph Ball and After Dairon - Central Unit
   > Dock to EMMI Zone Exit North
       Speed Booster and Open Charge Door
-  > Lower-Left Area
-      Flash Shift or Movement (Beginner) or Lay Cross Bomb or Use Spin Boost
 
 > Pickup (Energy Part); Heals? False
   * Layers: default
   * Pickup 63; Major Location? False
   * Extra - actor_name: item_energyfragment_000
   * Extra - actor_def: actordef:actors/items/item_energyfragment/charclasses/item_energyfragment.bmsad
-  > Tile Group (WEIGHT)
-      Morph Ball
-
-> Tile Group (POWERBEAM) 1; Heals? False
-  * Layers: default
-  * Configurable Node
-  * Extra - actor_name: breakabletilegroup_043
-  * Extra - actor_def: actordef:actors/logic/breakabletilegroup/charclasses/breakabletilegroup.bmsad
-  * Extra - actor_layer: breakables
-  * Extra - tile_types: ('POWERBEAM',)
-  > Tile Group (POWERBEAM) 2
-      Trivial
-  > Lower-Left Area
-      Trivial
-
-> Tile Group (POWERBEAM) 2; Heals? False
-  * Layers: default
-  * Configurable Node
-  * Extra - actor_name: breakabletilegroup_044
-  * Extra - actor_def: actordef:actors/logic/breakabletilegroup/charclasses/breakabletilegroup.bmsad
-  * Extra - actor_layer: breakables
-  * Extra - tile_types: ('POWERBEAM',)
-  > Dock to Big Hub
-      Trivial
-  > Tile Group (POWERBEAM) 1
-      Trivial
-
-> Tile Group (WEIGHT); Heals? False
-  * Layers: default
-  * Extra - actor_name: breakabletilegroup_057
-  * Extra - actor_def: actordef:actors/logic/breakabletilegroup/charclasses/breakabletilegroup.bmsad
-  * Extra - actor_layer: breakables
-  * Extra - tile_types: ('WEIGHT',)
   > Dock to EMMI Zone Exit North
       Trivial
 
 > Dock to EMMI Zone Exit North; Heals? False
   * Layers: default
   * Open Passage to EMMI Zone Exit North/Dock to Yellow EMMI Introduction
+  > Door to EMMI Zone Exit East
+      Speed Booster
   > Pickup (Energy Part)
       All of the following:
           Missiles
           Lay Bomb or Lay Power Bomb
-  > Lower-Left Area
-      Speed Booster
```

Comment:
This removes speed as a possibility to go back to big hub, so I would either connect this to there instead, or add the speed reqs from the comment above to big hub as well.